data-sources/logs-default/config.yml

```diff
@@ -0,0 +1,8 @@
+id: logs-default
```

Comment:
This is a broad article about the different ways to send logs to new relic. Consider as core data source?